translations/expansion_content.tex/pl.po

```diff
@@ -153,6 +163,12 @@ msgid ""
 "War machines are also more expensive at the Trading Post.\n"
 "\n"
 msgstr ""
+"\\subsection*{\\hypertarget{War Machines}{Machiny Wojenne}}\n"
+"Dodane w rozszerzeniu Bastion.\n"
+"Machiny Wojenne\\index{War Machines} są Kartami Stałymi, można je zakupić w \\hyperlink{Trading Post}{Punkcie Handlowym} lub \\hyperlink{War Machines}{Fabryce Machin Wojennych}.\n"
+"Jeśli kupisz jedną z nich w Punkcie Handlowym, \\textbf{nie możesz korzystać} z żadnych innych normalnych funkcji tego Pola podczas tej Wizyty.\n"
+"\\textbf{Uwaga!} W Punkcie Handlowym Maszyny Wojenne są droższe."
+"\n"
```

Comment:
To samo linijki 203-221. Tam jest opis do karty z balistą.

Reply:
Swoją drogą, możesz zeskanować balistę po polsku. Kółek nie trzeba będzie dodawać, bo tutaj dodaje je LaTeX.